sample-apps/python-manual-instrumentation-sample-app/app.py

```diff
@@ -0,0 +1,226 @@
+    unit='ms'
+        )
+
+# Test HTTP instrumentation
+@app.route("/outgoing-http-call")
+def call_http():
+    with tracer.start_as_current_span("outgoing-http-call") as span:
+
+        # Demonstrates setting an attribute, a k/v pairing.
+        span.set_attribute("operation.name", "outgoing-http-call")
+
+        # Demonstrating adding events to the span. Think of events as a primitive log.
+        span.add_event("Making a request to https://aws.amazon.com/")
+        with tracer.start_as_current_span("amazon-request") as child:
+            requests.get("https://aws.amazon.com/")
+
+            print("updating bytes sent & latency time...")
+            update_total_bytes_sent()
+            update_latency_time()
+
+            return app.make_response(
+                convert_otel_trace_id_to_xray(
+                    trace.get_current_span().get_span_context().trace_id
+                )
+            )
+
+# Test AWS SDK instrumentation
+@app.route("/aws-sdk-call")
+def call_aws_sdk():
+
+    with tracer.start_as_current_span("aws-sdk-call") as span:
+        span.set_attribute("operation.name", "aws-sdk-call")
+        print("updating bytes sent & latency time...")
+        update_total_bytes_sent()
+        update_latency_time()
+
+        span.add_event("listing s3 buckets")
+        client = boto3.client("s3")
+        client.list_buckets()
+
+        return app.make_response(
+            convert_otel_trace_id_to_xray(
+                trace.get_current_span().get_span_context().trace_id
+            )
+        )
+
+# when this sample-app is invoked either by itself or a different sample app
+@app.route("/outgoing-sampleapp")
+def invoke():
+    # Call sample apps
+    with tracer.start_as_current_span("outgoing-sampleapp-parent") as parent:
+        parent.set_attribute("operation.name", "outgoing-sampleapp-call")
+        ports = cfg.get("SampleAppPorts")
+        if ports:
+            for port in ports:
+                parent.add_event("Sampleapp detected. Generating nested span.")
+                with tracer.start_as_current_span("outgoing-sampleapp-child") as child:
+                    uri = f"http://127.0.0.1:{port}/outgoing-sampleapp"
+                    print("making a request to: " + uri)
+                    r = requests.get(uri)
+
+        # If no sample apps are defined in the config file the app makes a request to amazon.
+        else:
+            with tracer.start_as_current_span("outgoing-sample-app-no-ports") as child:
+                print("no ports configured. making a request to https://aws.amazon.com instead.")
+                requests.get("https://aws.amazon.com/")
+        
+    update_total_bytes_sent()
+    update_latency_time()
+    
+    return app.make_response(
+        convert_otel_trace_id_to_xray(
+            parent.get_span_context().trace_id
+        )
+    )
+
+# Test Root Endpoint
+@app.route("/")
+def root_endpoint():
+    return "<h1>App running!</h1>"
```

Comment:
```suggestion
    return "OK"
```

No need for HTML or much of a response here.